SwiftUI_DZ/SwiftUI_DZ/InviteView/InviteView.swift

```diff
@@ -111,27 +123,69 @@ struct InviteView: View {
         }
     }
 
+    private var loadingCircleView: some View {
+        Circle()
+            .trim(from: 0,to: inviteViewModel.progressLoading)
+            .stroke(.orange, style: StrokeStyle(lineWidth: 10,
+                                                lineCap: .round))
+            .frame(width: 120)
+            .rotationEffect(.degrees(Constants.rotationAngleDouble))
+            .padding(.bottom, 20)
+    }
+    
     private var getStartedButtonView: some View {
-        Button {
-            inviteViewModel.isTabViewShown = true
-        } label: {
-            Text(Constants.getStartedString)
-                .font(.system(size: Constants.sizeFontDouble))
-                .fontWeight(.bold)
-                .foregroundColor(.black)
+        ZStack {
+            Button {
+                //            inviteViewModel.isTabViewShown = true
+                withAnimation(.linear(duration: 2)) {
+                    inviteViewModel.isStartProgress.toggle()
+                    inviteViewModel.progressLoading = inviteViewModel.isStartProgress ? 0.0 : 1.0
+                    inviteViewModel.progressViewActivate()
+                }
+            } label: {
+                Text(Constants.getStartedString)
+                    .font(.system(size: Constants.sizeFontDouble))
+                    .fontWeight(.bold)
+                    .foregroundColor(.black)
+            }
+            .frame(
+                width: Constants.getStartedButtonWidthHeightValue.0,
+                height: Constants.getStartedButtonWidthHeightValue.1,
+                alignment: .center
+            )
+            .background().foregroundColor(.white)
+            .cornerRadius(Constants.getStartedCornerRadiusValue)
+            Rectangle()
+                .frame(width: Constants.getStartedButtonWidthHeightValue.0 / Constants.twoDouble, height: 70)
+                .padding(.trailing, 140)
+                .opacity(0.01)
+                .gesture(longGesture)
         }
-        .frame(
-            width: Constants.getStartedButtonWidthHeightValue.0,
-            height: Constants.getStartedButtonWidthHeightValue.1,
-            alignment: .center
-        )
-        .background().foregroundColor(.white)
-        .cornerRadius(Constants.getStartedCornerRadiusValue)
     }
-
+        
+    private var developerInfoText: some View {
+        Text(Constants.developString)
+            .foregroundColor(.cyan)
+            .opacity(inviteViewModel.isLongTap ? 1 : 0)
+    }
+    
+    private var longGesture: some Gesture {
+            LongPressGesture(minimumDuration: 2)
+                .onEnded { _ in
+                    inviteViewModel.isLongTap.toggle()
+                    DispatchQueue.main.asyncAfter(deadline: .now() + Constants.fiveDouble) {
+                        withAnimation {
+                            inviteViewModel.isLongTap.toggle()
+                        }
```

Comment:
```suggestion
                    DispatchQueue.main.asyncAfter(deadline: .now() + Constants.fiveDouble) {
                        withAnimation {
                            inviteViewModel.isLongTap.toggle()
                        }
```
это во вью модель лучше